### PR TITLE
Docs: Reorder types, add concrete examples

### PR DESCRIPTION
**Pull request by saerdnaer**

### CODE REVIEW

File: docs/TYPES.md [hunk 1]
```diff
@@ -12,34 +12,127 @@ Each triggered `track` can and must be of a certain type, to be properly display
   - [`traffic`](#traffic)
```
Comment (rafaelmaeuer) [inline docs/TYPES.md:12]: I would prefer the toc to match the order of contents:

  - [`music`](#music)
  - [`live`](#live)
  - [`news`](#news)
  - [`weather`](#weather)
  - [`traffic`](#traffic)
  - [`audio`](#audio)
  - [`commercial`](#commercial)
  - [`jingle`](#jingle)

**Review by rafaelmaeuer — CHANGES_REQUESTED** [review]
- structure: please update the toc
- content: @frytg any feedback?

File: docs/TYPES.md (hunk 1 above)
Comment (saerdnaer) [reply docs/TYPES.md:12]: Sure, but I'd like to get feedback on the new order (ordered by importance /semantically close types together)

File: docs/TYPES.md [hunk 2]
```diff
@@ -12,34 +12,127 @@ Each triggered `track` can and must be of a certain type, to be properly display
...
+
 ## `traffic`
 
 Similar to `news` and `weather` it can mark the beginning of a traffic announcement.
 
-## `weather`
+```json
+{
+  "event": "de.ard.eventhub.v1.radio.track.playing",
+  "type": "traffic",
```
Comment (frytg) [inline docs/TYPES.md:118]: should be `weather`, right?

**Comment by saerdnaer** [comment]
@frytg Passt die neue Reihenfolge so für dich?

File: docs/TYPES.md [hunk 3]
```diff
@@ -3,43 +3,140 @@
...
+```
+
+## `weather`
+
+Similar to `news` it marks the beginning of a weather segment.
+
+```json
+{
+  "event": "de.ard.eventhub.v1.radio.track.playing",
+  "type": "traffic",
```
Comment (saerdnaer) [inline docs/TYPES.md:107]:
```suggestion
  "type": "weather",
```

**Review by frytg — APPROVED** [review]

**Review by rafaelmaeuer — APPROVED** [review]
I would suggest to remove empty lines 14-18, but this can also be done later

**Comment by rafaelmaeuer** [comment]
@frytg for merging we need to figure out, how to allow external PRs to read from github secrets

**Comment by saerdnaer** [comment]
> I would suggest to remove empty lines 14-18, but this can also be done later

You shoud be able to edit the file at https://github.com/saerdnaer/ard-eventhub/edit/patch-1/docs/TYPES.md?pr=%2Fswrlab%2Fard-eventhub%2Fpull%2F73

**Pull request merged**